app/main.py

```diff
@@ -1,3 +1,63 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if not isinstance(other, Distance):
+            return Distance(
+                km=self.km + other
+            )
+        return Distance(
+            km=self.km + other.km
+        )
+
+    def __iadd__(self, other):
+        if not isinstance(other, Distance):
+            self.km += other
+            return self
+        self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        return Distance(
+            km=self.km * other
+        )
+
+    def __truediv__(self, other):
+        return Distance(
+            km=round((self.km / other), 2)
+        )
+
+    def __lt__(self, other):
+        if not isinstance(other, Distance):
+            return self.km < other
```

Comment:
parameter other can be only of type Distance, int or float. Consider adding check for parameter other to be instance of int or float. The same things in further methods.